pkg/encoder/encoder.go

```diff
@@ -491,6 +491,14 @@ func (p *CompactEncoder) EventToString(response *tetragon.GetEventsResponse) (st
 			event := p.Colorer.Blue.Sprintf("⁉️ %-7s", "tracepoint")
 			return CapTrailorPrinter(fmt.Sprintf("%s %s %s %s", event, processInfo, tp.Subsys, tp.Event), caps), nil
 		}
+	case *tetragon.GetEventsResponse_ProcessLsm:
+		lsm := response.GetProcessLsm()
+		if lsm.Process == nil {
+			return "", ErrMissingProcessInfo
+		}
+		processInfo, caps := p.Colorer.ProcessInfo(response.NodeName, lsm.Process)
+		event := p.Colorer.Blue.Sprintf("❓ %-7s", "LSM hook")
```

Comment:
This is just a proposition, since we are into choosing emoji ahah:

I would drop the "hook" part for brevity, maybe you can use the lock emoji for LSM? Not sure but this is what pops when you type "security" in the emojipedia https://emojipedia.org/search?q=security. I would avoid the police officer. 
```suggestion
		event := p.Colorer.Blue.Sprintf("🔒 %-7s", "LSM")
```